guardian/commercial-templates
# Remove `manual-inline` template

## What does this change?



Removes the `manual-inline` template which is being used by the `inline-merchandising` slot, aka `im`. For more info check the work that removes the `im` slot from the Commercial codebase https://github.com/guardian/commercial/pull/1516

The `manual-inline` is removed from Svelte SSR and legacy.

Checked in GAM Creatives the most recent creative that has been created or modified that uses the native template `manual-inline` which was on the 19th of April and ended 2nd of May.  

## How to test



Run templates locally and you shouldn't see `manual-inline` in SSR: Static which is a svelte template or in legacy.
